src/MLDatasets.jl

```diff
@@ -29,6 +29,11 @@ import CSV
 @lazy import HDF5="f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
 # @lazy import JLD2
 
+# Required for ImageNet
+@lazy import JpegTurbo="b835a17e-a41a-41e7-81f0-2f016b05efe0"   # Open jpg-files
+@lazy import ImageCore="a09fc81d-aa75-5fe9-8630-4744c3626534"   # Preprocessing
+@lazy import StackViews="cae243ae-269e-4f55-b966-ac2d0dc13c15"  # Batching of images
```

Comment:
I'm quite confident that this will hit the world-age issue. It's a very thin package so there's no need to worry about its latency.

```suggestion
import StackViews
```